msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -166,7 +323,7 @@ The query parameters are:
 
 To use this deep link with your bot, specify this as the URL target in your card's button or tap action through the `openUrl` action type.
 
-## Generate deep links to channel conversation
+### Generate deep links to channel conversation
```

Comment:
Should check with Saurabh to make sure this is true, but may be worth calling out in this section and "deep links to file in channel" that those are teams only and have no strongly typed capabilities for other hosts at this time.